exporters/tests/httpd_docker_with_labels/Makefile

```diff
@@ -0,0 +1,22 @@
+VERSION ?= latest
```

Comment:
this file was cited anywhere?

Reply:
Yes in the updated example in `ExporterCommittitme.md`:

```
git clone https://github.com/dora-metrics/pelorus
pushd pelorus/exporters/tests/httpd_docker_with_labels/
export CONTAINERFILE_LOCATION=./Dockerfile
```

Reply:
deleted